PennMobile/Auth/PennAuthRequestable.swift

```diff
@@ -43,8 +49,7 @@ extension PennAuthRequestable {
                 }
             } else {
                 UserDefaults.standard.setShibbolethAuth(authedIn: false)
-//                completionHandler(nil, nil, NetworkingError.authenticationError)
-                completionHandler(nil, nil, NetworkingError.noInternet)
```

Comment:
You mentioned to me in a previous comment to change this completionHandler to return a NetworkingError.other.
I just wanted to note that it was originally a .authenticationError before I commented it out.